src/codegen/browser/code/context.ts

```diff
@@ -0,0 +1,77 @@
+export interface Import {
```

Comment:
What do you think about `src/schemas/imports.ts` and the respective types? It was meant to be a foundation for the custom imports feature, but since that one is postponed, we can easily introduce changes to it, since it's only used in a few places.

Reply:
Those types do seem identical, but I'm going to keep it as-is for the time being. I think there's a value in keeping the codegen logic completely separate, e.g. a change in the schema won't ripple out into a bunch of changes in the codegen.